chirp/drivers/h777.py

```diff
@@ -665,3 +670,34 @@ class BFM4Radio(H777Radio):
     def match_model(cls, filedata, filename):
         # This model is only ever matched via metadata
         return False
+
+@directory.register
+class BF1901Radio(H777Radio):
+    VENDOR = "Baofeng"
+    MODEL = "BF-1901"
+    PROGRAM_CMD = b'PWPG970'
+    ALIASES = []
+
+    VALID_BANDS = (400000000, 520000000)
+    MAX_VOXLEVEL = 9
+
+    _has_fm = True
+    _has_sidekey = False
+    _has_scanmodes = True
+    _has_scramble = False
+
+    @classmethod
+    def match_model(cls, filedata, filename):
+        # This model is only ever matched via metadata
+        return False
+
+@directory.register
+class BF1904Radio(BF1901Radio):
+    VENDOR = "Baofeng"
+    MODEL = "BF-1904"
+    ALIASES = []
+
```

Comment:
Do you want separate power levels for this? I think this is the 10W unit? Not a huge deal, but it should be easy.

Reply:
Different power levels but same subset of features... what do you suggest?

Reply:
It would be better to have different power levels exposed by that driver so that when importing memories it selects the closest one. You can just do the same thing you did for the VALID_BANDS but with a list of power levels.

Reply:
Got it